src/content/blog/cheerp-typescript.mdx

```diff
@@ -0,0 +1,393 @@
+---
+title: "TypeScript integration in Cheerp"
+description: "A tour of the TypeScript integration features that are a part of Cheerp. Cheerp is a powerful C++ to JavaScript and WebAssembly compiler."
+pubDate: "2024-03-05"
+authors: ["chen"]
+---
+
+import ThreeDemo from "@/components/ThreeDemo.svelte"; 
+
+Over the past months, I have been working to add
+[TypeScript](https://www.typescriptlang.org/) integration to Cheerp. TypeScript
+is a popular programming language that adds *static type checking* to
+JavaScript.
+
+There is lots to see, so let's jump right in!
```

Comment:
A TLDR would be good. Perhaps some bullet points that act as a table of contents, like
```
- Convert C++ into TypeScript declaration files
- Convert TypeScript declaration files into C++ headers
```

Reply:
I think the index on the right serves this purpose, no reason to duplicate the index in text.